docs/setup-cluster/deploy-cluster/slurm/hpc-with-agent.rst

```diff
@@ -0,0 +1,153 @@
+cluster-wide shared directory (examples in this section use ``$HOME``), create an ``agent.yaml``
+file. Below is a minimal example using a resource pool named for the user (``$USER``) and
+``singularity`` as the container runtime platform. If configured using variables such as ``$HOME``,
+a single ``agent.yaml`` could be shared by all users.
+
+.. code:: yaml
+
+   master_host: master.mycluster.com
+   master_port: 8090
+   resource_pool: $USER
+   container_runtime: singularity
+
+There are several other settings commonly configured in the `agent.yaml` which are listed in the
+table below. For the full list of options, see :ref:`agent-config-reference`.
+
++----------------------------+----------------------------------------------------------------+
+| Option                     | Description                                                    |
++============================+================================================================+
+| ``image_root``             | To avoid multiple image downloads, configure an image cache as |
+|                            | per :ref:`singularity-image-cache`                             |
++----------------------------+----------------------------------------------------------------+
+| ``container_runtime``      | Instead of ``singularity``, you could specify ``podman`` as    |
+|                            | the container runtime.                                         |
++----------------------------+----------------------------------------------------------------+
+| ``security``               | Secure the communications between the master and agent using   |
+|                            | TLS. Configure the sections of the ``security`` block as per   |
+|                            | :ref:`tls`.                                                    |
++----------------------------+----------------------------------------------------------------+
+
+****************************************************
+ Start Per-User Agents to Provide Compute Resources
+****************************************************
+
+The user may then start one or more agents to provide resources to their resource pool using the
+agent.yaml configured above.
+
+In the command examples below, it is assumed that the agent.yaml for a given user is provided in
+`$HOME``. Paths may need to be updated depending on your local configuration.
+
+On Slurm, you can allocate resources with the ``srun`` or ``sbatch`` commands with the desired
+resource configuration options.
+
+.. code:: bash
+
+   srun --gpus=8 /usr/bin/determined-agent  --config-file $HOME/agent.yaml
+
+or
+
+.. code:: bash
+
+   sbatch -N4 --gpus-per-node=tesla:4  --wrap="srun /usr/bin/determined-agent  --config-file $HOME/agent.yaml"
+
+On PBS, you can launch the agent on multiple nodes with the qsub command.
+
+.. code:: bash
+
+   qsub -l select=1:ncpus=4 -- /opt/pbs/bin/pbsdsh -- /usr/bin/determined-agent --config-file $HOME/agent.yaml
```

Comment:
Likely this would launch the agent on just one node. Did you mean to go with cpus?

Reply:
True.  I'll change it to select=2:gpus=4 which would  be 2 nodes and a total of 8 gpus.